apps/dashboard/src/components/create-workflow-button.tsx

```diff
@@ -126,7 +134,7 @@ export const CreateWorkflowButton = (props: CreateWorkflowButtonProps) => {
                   <FormItem>
                     <FormLabel>Name</FormLabel>
                     <FormControl>
-                      <InputField>
+                      <InputField state={errors['name']?.message ? 'error' : 'default'}>
```

Comment:
Are we going to do this for every field ?

Reply:
updated the code to make our lives easier ;)